contrib/statistics-tex.py

```diff
@@ -22,6 +22,11 @@
 
 sys.dont_write_bytecode = True  # prevent creation of .pyc files
 
+number_convert = {
```

Comment:
Are the changes to the old script intentional? If not, don't bother, because we will delete it anyway. Just wanted to make sure that the changes should not go to the new code.